cmd/cmd.go

```diff
@@ -78,6 +78,20 @@ func addDatabase(c *cobra.Command) *cobra.Command {
 	return c
 }
 
+var extraIndexesCmd = &cobra.Command{
+	Use:   "extra-indexes index-1 [index-2 …]",
+	Short: "Creates extra indexes in the JSON fields",
```

Comment:
"JSON fields" não faz muito sentido no jargão do MongoDB, faz?

```suggestion
	Short: "Creates extra indexes in the company fields",
```